sdcm/sct_config.py

```
@@ -3967,15 +3967,41 @@ def verify_configuration_urls_validity(self):
+                            version=os_version,
+                            shape=shape_name,
+                        )
+                    )
+                self["oci_image_db"] = " ".join(resolved_images)
+                # NOTE: use default format version because platform images don't have scylla tags
+                self["user_data_format_version"] = "3"
+            else:
+                oci_region_names = self.get("oci_region_name") or []
+                if not isinstance(oci_region_names, list):
+                    oci_region_names = [oci_region_names]
+                for image, region in zip(oci_image_db.split(), oci_region_names):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)